src/Schemes/OpenInclusion/Morphisms/Types.jl

```diff
@@ -0,0 +1,25 @@
+export OpenInclusion
+
+########################################################################
+# OpenInclusion                                                        #
+#                                                                      #
+# This is a special type of morphism for affine schemes for open       #
+# inclusions of one into the other.                                    #
+########################################################################
+
+@attributes mutable struct OpenInclusion{DomainType, CodomainType, PullbackType}<:AbsSpecMor{DomainType, CodomainType, PullbackType, OpenInclusion, Nothing}
```

Comment:
You could document the type instead. Then `?OpenInclusion` returns something useful.